app/models/course.rb

```diff
@@ -149,13 +149,17 @@ def open_for_applications?
 
   def applications_open_from
     site_statuses
-      .open_for_applications
+      .findable_or_new
+      .with_vacancies
```

Comment:
> This means that no date is available if applications haven't opened yet.
The wrong date might show if some of the sites have no vacancies.
No date available if the site is new and not yet running.

Not reading last time or this time properly....

But I guess 

```
Get me the applications open from date from the course
where at least one site statuses is both
findable or new
with vacancies

and grab the earliest
```

hmmm 
why find or `new`?

Given that there `new` is just a transitional state that gets flipped when this is triggered?

https://github.com/DFE-Digital/manage-courses-backend/blob/6fb4326919b4ba1f1c51af633abcf87ee0ac8297/app/controllers/api/v2/courses_controller.rb#L37